molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
> That's the point.

Thats kind of the problem though.  People who use `molecule` may be developing a role which uses a callback plugin.  If we are setting `ANSIBLE_CALLBACK_PLUGINS` for idempotence checks, then the user may not be able to test their callback plugin.  If we do allow them to override it, then idempotence checks will not work.